### PR TITLE
🎅 Update repo metadata 

### PR DESCRIPTION
**Pull request by cthoyt**
This PR makes the following updates to OBO Foundry metadata:

1. Updates some out of date GitHub repository URLs (BCO, MP, NCRO)
2. Adds GitHub repositories to ontologies with no repository annotation (FBbt, FBdv)
3. Fixes repository annotations that weren't using GitHub (OARCS)
4. Update the contact's handle on TXPO to correspond to the contact person (to comply with [FP-11 - Locus of Authority](https://obofoundry.org/principles/fp-011-locus-of-authority.html)) instead of a group GitHub handle
5. Add several missing GitHub handles
6. Update MP's contact information to be a specific person (Sue Bello) instead of a contact list, which was also in violation of FP-11

### CODE REVIEW

File: ontology/nomen.md
Comment (matentzn) [inline ontology/nomen.md:6]: Wouldnt make that change.. The title is a bit at the discretion of the editors.

Comment (cthoyt) [reply ontology/nomen.md:6]: Alright will open a different issue for this. I am very against the redundant usage of acronyms in the title since it's wasting space and increasing possibility for confusion. For example, in the case of UBERON, there is a self-referential use of UBERON in the title.

Comment (cthoyt) [reply ontology/nomen.md]: Reverted in 4dad270

File: ontology/colao.md
Comment (matentzn) [inline ontology/colao.md:7]: Better not make that change before https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1695 is solved.

**Review by matentzn — CHANGES_REQUESTED** [review]

**Comment by matentzn** [comment]
Beautiful new years present, thanks @cthoyt and Happy New Year.

File: ontology/colao.md
Comment (cthoyt) [reply ontology/colao.md:7]: then should I change the other one in AISM to not have the accent?

Comment (cthoyt) [reply ontology/colao.md]: Oh that appears to be the solution referenced in the issue. Done in 7773c80

**Comment by cthoyt** [comment]
I'm done adding GitHub handles. The remaining active repositories without handles are mostly from the Rat Genome Database (RGD) crowd, whose repositories all seem to have been uploaded and edited by a small group of users whose handles don't obviously correspond to any of the listed names. Similar issues persisted for the remaining ones.

**Review by matentzn — APPROVED** [review]

File: ontology/clo.md
Comment (cthoyt) [reply ontology/clo.md:5]: Good thought, but I'd rather not get this PR tied up with information correctness. I'm preparing an automated script for contacting all OBO Foundry authors to check in and say if they're still in charge of the ontologies they're listed for or not for some future 2022 fun

**Comment by matentzn** [comment]
These changes are all great and pretty uncontroversial; unless there is any strong objections @nicolevasilevsky  could you please merge this on my behalf on Friday? No more changes required, all good (the one comment is for future work).

File: ontology/clo.md
Comment (matentzn) [reply ontology/clo.md:5]: Yeah, lets leave it to @yongqunh to change it on his own. Just noticed that.

**Comment by cthoyt** [comment]
Not a blocker for merge, but for general information, there only remain five "active" ontologies that don't yet have a GitHub handle associated with them:

| OBO ID   | Name                                            | Contact           | Email                   | Link                                 |
|----------|-------------------------------------------------|-------------------|-------------------------|--------------------------------------|
| aeo      | Anatomical Entity Ontology                      | Jonathan Bard     | J.Bard@ed.ac.uk         | https://obofoundry.org/ontology/aeo  |
| bto      | BRENDA tissue / enzyme source                   | Antje Chang       | a.chang@tu-bs.de        | https://obofoundry.org/ontology/bto  |
| fma      | Foundational Model of Anatomy Ontology (subset) | Onard Mejino      | mejino@u.washington.edu | https://obofoundry.org/ontology/fma  |
| opl      | Ontology for Parasite LifeCycle                 | Priti Parikh      | preets79@gmail.com      | https://obofoundry.org/ontology/opl  |
| mamo     | Mathematical modeling ontology                  | Nicolas Le Novere | n.lenovere@gmail.com    | https://obofoundry.org/ontology/mamo |

Side notes:

1. I have sent an email to all of these addresses asking for this information
2. I don't think @brenda-enzymes is valid under FP-11, which states that there should be a locus of authority - in other words, it's not sufficient IMO to have a handle that doesn't correspond to a specific person
3. For OPL I've already opened #1717 for discussion

**Review by matentzn — APPROVED** [review]

**Comment by cmungall** [comment]
Not sure aeo should be considered active. I can contact JB

**Comment by cthoyt** [comment]
@cmungall already done, see#1721

**Comment by cthoyt** [comment]
@nicolevasilevsky please merge #1725 before this one tomorrow since this PR duplicates what Meghan wrote there.

**Comment by nicolevasilevsky** [comment]
ok! Should I wait until tomorrow or merge now?

**Comment by cthoyt** [comment]
@nicolevasilevsky It'd be great to do now!

**Pull request merged**
